scripts/components/custom-select/custom-select.js

```diff
@@ -141,6 +141,7 @@ export const CustomSelect = (props) => {
 					MultiValueLabel: SortableMultiValueLabel,
 					Option: customOptionComponent ?? Option,
 					SingleValue: customSingleValueDisplayComponent ?? SingleValue,
+					IndicatorSeparator: () => null,
```

Comment:
Does this have to be a callback that isn't doing anything?

Reply:
Nope, copied it from the documentation 😅 
I'll change it